### ISSUE TITLE
URL in readme outdated

### ISSUE THREAD

**Issue by vmario89** [DDoSolitary/LxRunOffline#66]
Hi,
the readme links to an URL which does not exist anymore:
Configure default user, environment variables and "various flags". https://docs.microsoft.com/en-us/previous-versions/windows/desktop/api/wslapi/ne-wslapi-wsl_distribution_flags 

regards, Mario

**Comment by DDoSolitary** [comment]
Seems that functionality of the configuration flags have been surpassed (and possibly decrecated) by the `wsl.conf` file. I'll look into this.

**Comment by Biswa96** [comment]
It looks like WSL remove the "enumeration" entirely from [wslapi.h](https://docs.microsoft.com/en-us/windows/desktop/api/wslapi/) page. My speculative mind says that there will be more hidden flags :smiling_imp:

**Comment by Biswa96** [comment]
URL is fixed. New hidden flag is 8.

**Comment by DDoSolitary** [comment]
Yeah, you're right, but I'll keep this issue closed and reopen the corresponding issue in Microsoft/WSL
